### PR TITLE
add Slider page

### PR DESCRIPTION
**Pull request by hsnice16**
This PR will add a page for the Slider component in the development branch. 

This is how it looks,
image

Please review this PR, and give your feedback.

### CODE REVIEW

**Review by pradipgharge — APPROVED** [review]
Looks good to me, you can merge!

File: pages/components/slider.html [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+            <pre class="pre-code">
+              <code class="language-html"> 
+&lt;input
+  type="range"
+  min="1"
+  max="5"
+  value="2"
+  step="0.1"
+  class="slider"
+/>
```
Comment (prachi-sahani) [inline pages/components/slider.html:164]: suggestion - consider formatting your code

**Review by prachi-sahani — APPROVED** [review]
Code is good to merge! 👍🏻

**Review by Anurag021 — APPROVED** [review]
Code looks good to merge.

**Review by jalajcodes — APPROVED** [review]
Looks good to go. You can merge it.

File: pages/components/slider.html (hunk 1 above)
Comment (hsnice16) [reply pages/components/slider.html:164]: Thanks for checking this, Prachi. Actually, the formatting we do inside `<code>` tag in HTML file gets applied on the preview as well. And I don't want any space on the preview that's why. It is like that.

Like, it looks like this now 
image

And if I will add space in the HTML file, then the space will be visible here too. At this place, 
image
Which I don't want.

**Comment by hsnice16** [comment]
Thank you for reviewing this.

**Pull request merged**